doc/source/cluster/guide.rst

```diff
@@ -0,0 +1,277 @@
+and placement groups resource demands from the cluster, and try to add the
+minimum list of nodes that can fulfill these demands. When worker nodes are
+idle for more than :ref:`idle_timeout_minutes
+<cluster-configuration-idle-timeout-minutes>`, they will be removed (the head
+node is never removed unless the cluster is torn down).
+
+Autoscaler uses a simple binpacking algorithm to binpack the user demands into
+the available cluster resources. The remaining unfulfilled demands are placed
+on the smallest list of nodes that satisfies the demand while maximizing
+utilization (starting from the smallest node).
+
+**Here is "A Glimpse into the Ray Autoscaler" and how to debug/monitor your cluster:**
+
+2021-19-01 by Ameer Haj-Ali, Anyscale Inc.
+
+.. youtube:: BJ06eJasdu4
+
+
+Deploying an application
+------------------------
+
+The recommended way of connecting to a ray cluster is to use the
+``ray.client.connect()`` API and connect via the Ray Client.
+
+.. note::
+
+  Using ``ray.client.connect()`` is generally a best practice because it allows
+  you to test your code locally, and deploy to a cluster with **no code
+  changes**.
+
+To connect via Ray Client, set the ``RAY_ADDRESS`` environment variable to the
+address of the ray client server.
+
+:ref:`Learn more about setting up the ray client server here <ray-client>`.
+
+.. note::
+
+  When deploying an application, the job will be killed if the driver
+  disconnects.
+
+  A detached actor can be used to avoid having a long running driver.
+
+Monitoring and observability
+----------------------------
+
+Ray comes with 3 main observability features:
+
+1. :ref:`The dashboard <ray-dashboard>`
+2. :ref:`ray status <monitor-cluster>`
+3. :ref:`Prometheus metrics <multi-node-metrics>`
+
+Monitoring the cluster via the dashboard
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:ref:`The dashboard provides detailed information about the state of the cluster <ray-dashboard>`,
+including the running jobs, actors, workers, nodes, etc.
+
+By default, the cluster launcher and operator will launch the dashboard, but
+not publicly expose it.
+
+If you launch your application via the cluster launcher, you can securely
+portforward local traffic to the dashboard via the ``ray dashboard`` command
+(which establishes an SSH tunnel). The dashboard will now be visible at
+``http://localhost:8265``.
+
+With the kubernetes operator, you will need to expose port 8265 on the head
+node, or use `kubectl to portforward
+<https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/>`__.
+
+Observing the autoscaler
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Recall that the autoscaler makes decisions by considering the user defined
+configuration, scheduling information, and programatic information from the
+cluster. The autoscaler publishes its inputs from the cluster, the state of
+nodes being launched, and any errors generated, which can be accessed via the
```

Comment:
Could split the sentence up a bit

"; all of this information can be accessed with..."